### ISSUE TITLE
Can't delete users in postgres db with 2.5

### ISSUE THREAD

**Issue by wiad** [grafana/grafana#3216]
Upgraded to 2.5 and deletion of users no longer works with postgres as db backend. 

Outtake from grafana.log:
```
2015/11/12 08:52:21 [admin_users.go:95 AdminDeleteUser()] [E] Failed to delete user: pq: syntax error at or near "user"
2015/11/12 08:52:21 [I] Completed /api/admin/users/4 500 Internal Server Error in 3.516326ms
```

Postgres logs show:
```
2015-11-12T08:52:21.588888+00:00 lxserv266 WARNING [5-1] pgpid=1614 pguser=elinusr pghost=172.29.0.163 pgdb=grafana pgapp=[unknown] ERROR:  syntax error at or near "user" at character 13
2015-11-12T08:52:21.589309+00:00 lxserv266 WARNING [5-2] pgpid=1614 pguser=elinusr pghost=172.29.0.163 pgdb=grafana pgapp=[unknown] STATEMENT:  DELETE FROM user WHERE id = $1
```

I suspect this is because 'user' in postgres is a reserved keyword, so all operations towards a table called 'user' must be qouted, as such:
```
 DELETE FROM "user" WHERE id = $1
```

**Comment by torkelo** [comment]
this is already fixed in master, and in 2.5.1-pre1 builds http://grafana.org/download/builds.html 
https://github.com/grafana/grafana/commit/2676f24e0a09040331d49c391cb774ca447a8b11

**Comment by utkarshcmu** [comment]
This is fixed in #3157 and merged into master.

**Comment by wiad** [comment]
ah, guessed I only searched in 'open' issues. Sorry!
